studio/schemas/documents/compensations.ts

```diff
@@ -1,16 +1,28 @@
 import { defineField, defineType } from "sanity";
 import { titleSlug } from "../schemaTypes/slug";
 import seo from "../objects/seo";
+import locations from "../objects/locations";
 import { title } from "../fields/text";
 import { benefitId } from "./benefit";
 
+/*
+ maximum number of locations without preview string shortening
+ (should be at least 2)
+ given array: ["Trondheim", "Oslo", "Bergen", "Molde"]
+ 2 => Trondheim, and more
+ 3 => Trondheim, Oslo, and more
+ 4 => Trondheim, Oslo, Bergen, and Molde
+*/
```

Comment:
I don't see my comment anymore, but:

```suggestion
export const compensationsId = "compensations";

// Defines the maximum number of office locations to display in the preview without truncating
const OFFICES_PREVIEW_CUTOFF = 3;
```

Reply:
If we want consistent sorting it should be integrated with Prettier. 

How do you handle `export default` if `export` should come after `import`s?